pkg/autoscaler/http_scrape_client.go

```diff
@@ -68,8 +68,8 @@ func extractData(body io.Reader) (*Stat, error) {
 	for m, pv := range map[string]*float64{
 		"queue_average_concurrent_requests":         &stat.AverageConcurrentRequests,
 		"queue_average_proxied_concurrent_requests": &stat.AverageProxiedConcurrentRequests,
-		"queue_operations_per_second":               &stat.RequestCount,
-		"queue_proxied_operations_per_second":       &stat.ProxiedRequestCount,
+		"queue_requests_per_second":                 &stat.RequestCount,
```

Comment:
Does this break previous clients somehow? Can we just replace those or we need to have a version where we report both?
@mattmoor, what's your take?

Reply:
When upgrading knative version, I think controller will update all revision pods with the new queue-proxy image. No?

Reply:
what I am more concerned about is that existing dashboards and perhaps monitoring that we/users haves setup will stop working.

Reply:
We don't have dashboards but users may have set up their owns. I can keep `queue_operations_per_second`.